src/blocks/cpu.rs

```diff
@@ -201,11 +201,11 @@ impl CpuTime {
         let idle = u64::from_str(s.next()?).ok()?;
         let iowait = u64::from_str(s.next()?).ok()?;
         let irq = u64::from_str(s.next()?).ok()?;
-        let softirq = u64::from_str(s.next()?).ok()?;
+        let soft_irq = u64::from_str(s.next()?).ok()?;
 
         Some(Self {
             idle: idle + iowait,
-            non_idle: user + nice + system + irq + softirq,
+            non_idle: user + nice + system + irq + soft_irq,
```

Comment:
I'd leave softirq as written. See for example:

https://archive.kernel.org/oldlinux/htmldocs/kernel-hacking/basics-softirqs.html

Reply:
Thanks the extra context!